libmscore/score.cpp

```diff
@@ -2887,24 +2887,37 @@ void Score::padToggle(Pad n, const EditData& ed)
       std::vector<ChordRest*> crs;
 
       if (selection().isSingle()) {
-            // do not allow to add a dot on a full measure rest
-            Element* e = selection().element();
-            if (e && e->isRest()) {
-                  Rest* r = toRest(e);
-                  if (r->isFullMeasureRest())
-                        _is.setDots(0);
-                  }
 
-            // on measure rest, select the first actual rest
-            ChordRest* cr = selection().cr();
-            if (cr && cr->isRest() && cr->measure()->isMMRest()) {
-                  Measure* m = cr->measure()->mmRestFirst();
-                  if (m)
-                        cr = m->findChordRest(m->tick(), 0);
-                  }
+            Element* selectedElement = selection().element();
+
+            if (!selectedElement->isNote()) {
+
+                  ed.view->startNoteEntryMode();
+                  deselect(selectedElement);
+
+                  } else {
```

Comment:
This doesn't follow the coding principal, move `else {` to a new line.

Reply:
thx, fixed

Reply:
You can mark these conversations as resolved so the outdated code and the conversation can be folded.